csvimport.json

```diff
@@ -0,0 +1,215 @@
+{
+    "appid": "1b0e5665-faeb-4c8e-9fd3-8627cd90d530",
+    "name": "CSV Import",
+    "description": "Ingest CSV files into Phantom",
+    "type": "information",
+    "product_vendor": "Splunk",
+    "logo": "logo_splunk.svg",
+    "logo_dark": "logo_splunk_dark.svg",
+    "product_name": "CSV",
```

Comment:
Shouldn't this be `CSV Import`? @dgopani-crest, any thoughts on this?

Reply:
yes we can